### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by raskhodchikov**
Fixes for:
- The .sln should be at the root of the project to make it easier for the user to import the project
- AppConfig constants should be inlined and used as raw strings.
- The NotifyEvent body is not passed correctly
- change the RunnerAction in code
- The RunnerActions class is quite different than the other languages
- The config value for "tdl_require_rec" will be "true" (lowercase). This needs to be reflected in the IsRecordingSystemOk default and in the way we parse the string
- The paths "..\..\..\..\config\credentials.config" do not resolve correctly on OSX/Linux
- If no file is present, the GetLastFetchedRound fails with an exception

### CODE REVIEW

File: src/BeFaster/Runner/CredentialsConfigFile.cs [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
     {
         private static readonly Dictionary<string, string> Properties = new Dictionary<string, string>();
 
-        static CredentialsConfig()
+        static CredentialsConfigFile()
         {
             try
             {
-                foreach (var row in File.ReadAllLines(AppConfig.CredentialsFilePath))
+                foreach (var row in File.ReadAllLines(@"..\..\..\..\config\credentials.config"))
```
Comment (julianghionoiu) [inline src/BeFaster/Runner/CredentialsConfigFile.cs]: Is this path going to resolve correctly on a Non-Windows environment.
Do we need to use`Path.Combine` here too?

Comment (raskhodchikov) [reply src/BeFaster/Runner/CredentialsConfigFile.cs]: Fixed for path.combine

**Comment by julianghionoiu** [comment]
I have tested on both VS 2017 and Rider and things look better now.
I have push another commit with a Path fix. Can you check if it still works on your machine?

While testing I have noticed another issue: Waiting for requests takes to long to exit. The timeout value should be interpreted as miliseconds rather than seconds.

Another pending task is to update the top level README file so that it specifies the requirements like minimum version of .Net and the next step to follow.

**Comment by raskhodchikov** [comment]
Tested on my machine, it's working.
Fixed timeout value in client, updated version here and updated readme file.

**Pull request merged**